demos/k8s-demo/README.md

```diff
@@ -6,15 +6,15 @@ It's intended for advanced users and is light on explanations.
 
 For a friendlier version of this tutorial, with breakdowns of every step, try:
 
-[Our Detailed Introduction to Secretless in Kubernetes](https://secretless.io/docs/get_started/kubernetes_tutorial.html)
+[Our Detailed Introduction to Secretless in Kubernetes](https://secretless.io/docs/tutorials/kubernetes/kubernetes-tutorial-base.html)
```

Comment:
can we get this out of the `docs` folder? it seems odd now to have that on the path